src/conn/conn_compact.c

```diff
@@ -402,6 +402,10 @@ __wt_background_compact_end(WT_SESSION_IMPL *session)
           session, background_compact_ema, conn->background_compact.bytes_rewritten_ema);
     }
 
+    __wt_verbose_info(session, WT_VERB_COMPACT,
```

Comment:
We should use the category `WT_VERB_COMPACT_PROGRESS` here instead. Only for the reason that the `WT_VERB_COMPACT_PROGRESS` is logged at info level to MDB by default, whereas `WT_VERB_COMPACT` would require a config change.

Reply:
Thanks for the suggestion! a7dbb614e2cd0c78872fab8bcb07b87933ac48b4